### ISSUE TITLE
Status bar broken if status bar item has to long text

### ISSUE THREAD

**Issue by evidolob** [containers/podman-desktop#1723]
### Bug description

If I add `StatusBarItem` with "long" `text` , it break status bar alignment:

<img width="793" alt="Screenshot 2023-03-16 at 15 16 40" src="https://user-images.githubusercontent.com/929743/225628653-144e4bbf-40f3-4472-b437-9f2d4dc9bc24.png">

Not sure how long text string start breaking the alignment, it depends on window width.

### Operating system

 macOS 13.2.1 Intel

### Version

0.12.0

### Steps to reproduce

Just put this sample in your extension:

```typescript
  const testItem = extensionApi.window.createStatusBarItem('RIGHT');
  testItem.text = 'Some veryyyyyyyyyyyyyyyyyyyyyyyy looooooooooooooooooooong text';
  testItem.show();
```

And look on status bar behaviour, if its look normally, try to shrink the app window.

**Comment by deboer-tim** [comment]
This is because each 'side' of the status bar gets 1/2 width, then it wraps. We should fix this, but... why are you hitting it? Status bar items should normally be very short, like icons or single words.

**Comment by evidolob** [comment]
I use status bar to indicate process of downloading crc installer and bundle. 
As for now, I don't have a better option.

**Comment by gbraad** [comment]
> like icons or single words.

due to limitations in the API we have no other place to place the download progress. We might be able to abbreviate as `D/L 67%`, but this also for decompression, etc.

I hope what can be done is to allow any size of string, but show it as a marquee scrolling text.

**Comment by deboer-tim** [comment]
Fixed via #1765.
